Gemfile

```diff
@@ -31,6 +30,7 @@ group :development, :test do
   gem 'capybara-webkit'
   gem 'database_cleaner'
   gem 'factory_bot'
+  gem 'foreman'
```

Comment:
The commit talks about removing `foreman`, but in this case you're still keeping it.  Why is it still required here?

Reply:
Ah probably copy and pasting identical messages fail. I kept it in dev so it could still be run in https://github.com/alphagov/govuk-puppet/blob/master/development-vm/Procfile but I've since updated that so can now remove it. Thanks